src/SFA.DAS.EmployerIncentives.Domain/NewApprenticeIncentive/NewApprenticeIncentive.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using SFA.DAS.EmployerIncentives.Abstractions.Domain;
+using SFA.DAS.EmployerIncentives.ValueObjects;
+
+namespace SFA.DAS.EmployerIncentives.Domain.NewApprenticeIncentive
+{
+    public class NewApprenticeIncentive : AggregateRoot
```

Comment:
As mentioned above shouldn't this be a value object and inherit from SFA.DAS.EmployerIncentives.Abstractions.Domain.ValueObject?